text/0985-v2-addon-by-default.md

```diff
@@ -0,0 +1,81 @@
+---
+stage: accepted
+start-date: 2023-11-02T16:05:11.000Z 
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/985
+project-link:
+suite: 
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+# Change the default addon blueprint
+
+## Summary
+
+`@embroider/addon-blueprint` has been in progress for a while, it's good stuff. 
+
+
+## Motivation
```

Comment:
motivation for having external blueprint, rather than built in to ember-cli?
- easiest to test
- ember-cli's test suite is already quite slow (due to tons of tests)
- can be independently versioned, patched, etc